### PR TITLE
Make READMEs even more obviously versioned.

### PR DESCRIPTION
**Pull request by thockin**
@brendandburns a start

### CODE REVIEW

**Comment by thockin** [comment]
@zmerlynn

File: docs/README.md [hunk 1]
```diff
@@ -1,29 +1,44 @@
...
-Documentation for previous releases is available at
-http://releases.k8s.io/vX.Y.Z/docs.
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+If you are using a released version of Kubernetes, you almost certainly want
+the docs that go with that version.
+
+Documentation for specific releases is available at
+releases.k8s.io/v*X*.*Y*.*Z*/docs
+   * [releases.k8s.io/v0.21.0](http://releases.k8s.io/v0.21.0/docs)
```
Comment (zmerlynn) [inline docs/README.md]: Is anything supposed to be keeping this list up to date?

**Comment by zmerlynn** [comment]
LGTM.

File: docs/README.md (hunk 1 above)
Comment (thockin) [inline docs/README.md]: Not really.  Open for discussion, I guess.  It's more like "here are some
examples".

Comment (zmerlynn) [inline docs/README.md]: No worries, just curious. We could point it to the releases page on GitHub and fix up the docs links off there.

**Comment by brendandburns** [comment]
LGTM.  Although I'd vote for making the warning bold face. or even <blink>

**Comment by thockin** [comment]
Github won't allow most HTML

**Comment by thockin** [comment]
No1 100% bigger and bolder.  Would that I could blink or change colors, but alas GitHub's markdown makes a poor document-publishing platform.

**Comment by thockin** [comment]
Needs an ok-to-merge

**Comment by thockin** [comment]
self-OK'ing and merging - progress.

**Pull request merged**